README.md

```diff
@@ -1,2 +1,60 @@
-# object_detection_example_structure
-Example of folder structure of a complete run on DeepSportRadar Instance Segmentation Challenge dataset
+### Object Detection example on a selection on SportRadar Challenge
+
+This is an example of folder structure in the Lightly format for data selection using data from the [DeepSportRadar Instance Segmentation Challenge](https://github.com/DeepSportRadar/instance-segmentation-challenge)!
+Predictions are computed on the union of val and test sets using a detectron2 mask_rcnn_R_50_FPN_3x model finetuned on the train set. Metadata are created with the name of the folders in which images are contained. KS-FR-ROANNE folder is removed using our [relevant filenames feature](https://docs.lightly.ai/docs/relevant-filenames).
```

Comment:
I didn't find any mention of `KS-FR-ROANNE` in the relevant filenames file.

Reply:
I canceled the KS-FR-ROANNE from the relevant filename list, which comprehends every filename but the KS-FR-ROANNE ones. Isn't it the proper way to do that?